drivers/unix/net_socket_posix.cpp

```diff
@@ -317,7 +317,7 @@ Error NetSocketPosix::bind(IP_Address p_addr, uint16_t p_port) {
 Error NetSocketPosix::listen(int p_max_pending) {
 	ERR_FAIL_COND_V(!is_open(), ERR_UNCONFIGURED);
 
-	if (::listen(_sock, p_max_pending) == SOCK_EMPTY) {
+	if (::listen(_sock, p_max_pending) == (int)SOCK_EMPTY) {
```

Comment:
Same as above, make this `::listen(...) != 0`.

Reply:
Fixed.